libs/ledger-live-common/src/families/cosmos/specs.ts

```diff
@@ -430,17 +431,28 @@ function cosmosLikeMutations(minimalTransactionAmount: BigNumber): MutationSpec<
   ];
 }
 
-const generateGenericCosmosTest = (currencyId: string, config?: Partial<AppSpec<Transaction>>) => {
+const generateGenericCosmosTest = (
+  currencyId: string,
+  isExpertModeRequired: boolean,
+  config?: Partial<AppSpec<Transaction>>,
+) => {
   return {
     name: currencyId,
     currency: getCryptoCurrencyById(currencyId),
     appQuery: {
       model: DeviceModelId.nanoS,
       appName: "Cosmos",
+      appVersion: "2.34.11",
```

Comment:
We want to remove this now, right?

Reply:
i'm going to try while it's getting qaed

Reply:
Done